package.json

```diff
@@ -28,7 +28,17 @@
         "pretty-quick": "3.1.3",
         "rimraf": "5.0.1",
         "ts-jest": "29.1.0",
-        "typescript": "4.9.4"
+        "typescript": "4.9.4",
```

Comment:
Could you please explain why these packages are added to the root package.json? Please put them where they are used. In addition, please keep your esbuild configs in the `control-property-editor` project since it is the only one using it. If we use it in more place, we can extract what we need.

Reply:
Just noticed that you use it in two modules. Let's wait for the discussion about private modules. If the outcome is to keep the flat structure then keep the esbuild config in root.
Nevertheless, some of the packages need an explanation e.g. `express` and `react`

Reply:
also it was based on the suggestion from Austin on slack(os channel) 

>  I would suggest moving packages/control-property-editor/esbuildConfig.js to esbuildConfig.js . that we will start to use esbuild in other areas on open-ux-tools soon.